docs/lib/project-setup/fragments/flutter/create-application/10_createProject.md

```diff
@@ -14,6 +14,20 @@ Next, configure your project:
 
 Android Studio will open your project with a tab opened to *main.dart*
 
-Lastly, modify your Podfile to target iOS platform 11.0 or higher.  Within your project open `ios/Podfile` and change the second line to be `platform :ios, '11.0'. 
+If you use CocoaPods, modify your Podfile to target iOS platform 11.0 or higher.  Within your project open `ios/Podfile` and change the second line to be `platform :ios, '11.0'. 
+
+<details>
+<summary>
+CocoaPods instructions
+</summary>
+
+```bash
+sudo gem install cocoapods
+
+# make sure you are in the project root (the ios folder)
+pod init
+```
+
+</details>
```

Comment:
unsure if cocoapods is a hard dependency - in anycase, podfile isnt there by default so i added it